### PR TITLE
[scd] Store past OVNs on operational intents upsert

### PR DESCRIPTION
**Pull request by mickmis**
PR stacked on top of #1095 and #1100: please only consider beb2042b3308c689bc7b27d572fb23dc81de4dbe for review.
Continuation of #1078 

This enables storing all past OVNs in the DB.

### CODE REVIEW

File: pkg/scd/operational_intents_handler.go [hunk 1]
```diff
@@ -365,7 +365,7 @@ func (a *Server) UpdateOperationalIntentReference(ctx context.Context, req *rest
 
 type validOIRParams struct {
 	id             dssmodels.ID
-	ovn            restapi.EntityOVN
+	ovn            scdmodels.OVN
```
Comment (Shastick) [inline pkg/scd/operational_intents_handler.go:368]: Thanks, that's a convenient change

Comment (Shastick) [reply pkg/scd/operational_intents_handler.go:368]: How do we/will we differentiate between a client specifying an OVN as part of a 'normal' mutation (where the current OVN needs to be provided)?

(I see that #1078 mentions a new `requested_ovn_suffix` parameter which I assume will be added in upcoming PRs?)

**Review by Shastick — APPROVED** [review]
PR looks good to me (review was limited to commit `4a3db8c`)

**Comment by Shastick** [comment]
Side thought: if the parent PR still needs a bit of work, the current refactoring effort (#1088) would benefit from the change of `ovn` to `scdmodels.OVN`. Could you consider splitting that to a separate PR?

(Otherwise I could also do so)

File: pkg/scd/operational_intents_handler.go (hunk 1 above)
Comment (mickmis) [reply pkg/scd/operational_intents_handler.go:368]: > (I see that https://github.com/interuss/dss/issues/1078 mentions a new requested_ovn_suffix parameter which I assume will be added in upcoming PRs?)

Exactly

**Pull request merged**